### ISSUE TITLE
Disposals Travel and Plasma?

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#18590]
(WEB REPORT BY: shadownil REMOTE: 172.93.109.202:7777)
# Revision
032daef729f24d906d8257b818ebb609e352c0bf

# Description
Using Disposals to travel through rooms is all fine and good, but if you go through a room with plasma in it while in disposals, you still get plasma'd.

# Steps to Reproduce
Have any room that the disposals route goes through be filled with purple snow. (Malf Rounds make this easy). Disposals yourself up the route so you end up going through that room on the way.

# What you Expected
To be safe in the disposals pipes and be free of purple snow.

# What Actually Happened
All my clothing and body became plasma'd.

**Comment by Pathid** [comment]
Yeah, disposals don't have their own environment separate from the air above. But even if they _did_, it wouldn't make sense for a pneumatic mail system to have a nice livable 101.3 kPa of air either.

**Comment by Exxion** [comment]
I'm about 90% sure that the disposals container objects *do* have their own air

**Comment by Pathid** [comment]
Right, I'm a bozo. The _holders_ do. Probably just need a ``return_air()`` override or something.
